docs/workflows/data/table-delete-row.md

```diff
@@ -17,4 +20,4 @@ There are four options for defining what rows to delete:
 * **Match all of the following**: Allows [building a query](/data-tables/overview/#querying-table-data) where rows that match **all** of the column conditions will be deleted.
 * **Advanced**: Allows [building an advanced query](/data-tables/overview/#advanced-queries) using a [JSON template](/workflows/accessing-payload-data/#json-templates). Rows that match the query will be deleted.
 
-Optionally, the node also takes a [payload path](/workflows/accessing-payload-data/#payload-paths) to store the result of the deletion. If the node successfully found and deleted rows an object of the form `{ result: true, count: <count> }` will be placed at the payload path, where count is the number of rows deleted. If there was an error, the value at the payload path will be an error object. In the example above, this value will be placed at `data.deletionResult`.
\ No newline at end of file
+Optionally, the node also takes a [payload path](/workflows/accessing-payload-data/#payload-paths) to store the result of the deletion. If the node successfully found and deleted rows an object of the form `{ result: true, count: <count> }` will be placed at the payload path, where count is the number of rows deleted. If there was an error, the value at the payload path will be an error object. In the example above, this value will be placed at `data.deletionResult`.
```

Comment:
" If the node successfully found and deleted rows an object of the form `{ result: true, count: <count> }` will be placed at the payload path, where count is the number of rows deleted."

I stumbled through this sentence. Could just be me, but maybe a a comma after "deleted rows,"